libs/s25main/ingameWindows/IngameWindow.h

```diff
@@ -75,6 +73,8 @@ class IngameWindow : public Window
     void MouseLeftUp(const MouseCoords& mc);
     void MouseMove(const MouseCoords& mc);
 
+    GUI_ID GetID() const { return static_cast<GUI_ID>(Window::GetID()); }
```

Comment:
what if its not a GUI_ID?

Reply:
I renamed it to GetGUIID, good suggestion.